pandas/core/frame.py

```diff
@@ -1247,6 +1247,10 @@ def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
         encoding : string, optional
             A string representing the encoding to use in the output file,
             defaults to 'ascii' on Python 2 and 'utf-8' on Python 3.
+        compression : string, optional
+            a string representing the compression to use in the output file, 
+            currently only supports gzip and bz2 (see core.common._get_handle), 
```

Comment:
allowed values are 'gzip', 'bz2' (don't reference ``_get_handle`` that's an internal routine)